tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -166,6 +180,10 @@ def _return_class_from_tap(self, return_class='ContactDetails'):
                 return EditContact(self.marionette)
             elif return_class == 'SelectContact':
                 return None
+            elif return_class == 'SetIceContacts':
```

Comment:
That does not work there, at that point the screen shown should be the the ice contacts view in contacts settings